xml/html_title.xml

```diff
@@ -331,4 +331,16 @@
     <param pos="0" name="os.product" value="Linux"/>
     <param pos="0" name="os.cpe23" value="cpe:/o:debian:debian_linux:-"/>
   </fingerprint>
+  <fingerprint pattern="^Eagle Eye Network">
+    <description>EagleEyeNetworks to manage IoT cameras unauthorized webpage</description>
+    <example>Eagle Eye Networks (Local)</example>
+    <param pos="0" name="service.product" value="Cloud Video Surveillance System"/>
```

Comment:
Why service?  I think `hw` is more appropriate here -- the service is likely nginx/apache/whatever and not something eagle-eye specific.

Reply:
addressed in https://github.com/rapid7/recog/pull/235/commits/41ea23dc54f6c0553c03c8055d97bfce7d0a0da7